scale/docs/rest/v6/job_type.rst

```diff
@@ -132,76 +146,344 @@ The services will be replaced as the new v6 job type services are created:
+| revision_num             | Ingeger           | The number of versions of this job type.                                 |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| docker_image             | String            | The Docker image containing the code to run for this job.                |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| manifest                 | String            | Seed manifest describing Job, interface and requirements.                |
+|                          |                   | (See :ref:`architecture_seed_manifest_spec`)                             | 
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| configuration            | JSON Object       | JSON description of the configuration for running the job                |
+|                          |                   | (See :ref:`architecture_jobs_job_configuration_spec`)  		          |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| created                  | ISO-8601 Datetime | When the associated database model was initially created.                |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| deprecated               | ISO-8601 Datetime | When the job type was last deprecated (archived).                        |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| paused                   | ISO-8601 Datetime | When the job type was last paused.                                       |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+| last_modified            | ISO-8601 Datetime | When the associated database model was last saved.                       |
++--------------------------+-------------------+--------------------------------------------------------------------------+
+
+.. _rest_v6_job_type_revisions:
+
+v6 Job Type Revisions
+---------------------
+
+**Example GET /v6/job-types/{name}/{version}/revisions/ API call**
+
+Request: GET http://.../v6/job-types/{name}/{version}/revisions/
+
+Response: 200 OK
+
+ .. code-block:: javascript  
+    { 
+        "count": 1, 
+        "next": null, 
+        "previous": null, 
+        "results": [ 
+            { 
+        		"id": 3, 
+        		"job_type": { 
+                    "id": 1,
+                    "name": "my-job",
+                    "title": "My first job",
+                    "description": "My very first job",
+                    "icon_code": 012F
```

Comment:
Use the same job type serializer here as revision details so version info is included in the response.